fees/predy-finance/index.ts

```diff
@@ -309,10 +428,16 @@ const adapter: BreakdownAdapter = {
     //     start: async () => 1671092333,
     //   },
     // },
-    v320: {
+    // v320: {
```

Comment:
could you set like this is this version is not support 
https://github.com/DefiLlama/dimension-adapters/blob/master/protocols/pancakeswap/index.ts#L120

Reply:
@dtmkeng Thank you for reviewing, I've disabled v3 and v3.2.